### ISSUE TITLE
Are templates not implemented yet?

### ISSUE THREAD

**Issue by CIAvash** [MARTIMM/perl6-gnome-gtk3#26]
I see `init-template` and `class-set-template-from-resource` in `Widget.pm6`, but are all commented.

**Comment by MARTIMM** [comment]
Yes, you are right. It is an interesting part to implement but I was postponing everything around extending widgets to a later moment until I am confident enough. Now, I feel that I put my foot in a pit of vipers, so to speak.  I will keep this issue open so you can see when I get to it.
